### PR TITLE
SPCTR-3024 Fix: Buttons - Resolved an issue where child responsive padding was not working as expected on the frontend.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3024

### Screenshots

- https://share.getcloudapp.com/7KuWb8q9

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Added to the card.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by aapatre — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -166,6 +166,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ## Changelog ##
 
+### x.x.x ###
+* Fix: Buttons - Resolved an issue where child responsive padding was not working as expected on the frontend.
```
Comment (bsf-zanev) [inline README.md:170]:
```suggestion
* Fix: Buttons - Resolved an issue wherein the child responsive padding would not work as expected on the frontend.
```

**Review by bsf-zanev — APPROVED** [review]
Looks good, just a changelog update needed.

**Review by adi3890 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (AbhijitNage123) [reply README.md]: Done

Comment (AbhijitNage123) [reply README.md]: Done

**Pull request merged**